### PR TITLE
Update Pycharm EAP (CE & Pro) (2016.1.1 EAP, build 145.256.43)

### PR DESCRIPTION
**Pull request by tjanson**
I hope the version string is fine like this?

### CODE REVIEW

File: Casks/pycharm-ce-eap.rb [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-  url "https://download.jetbrains.com/python/pycharm-community-#{version}.dmg"
+  url "https://download.jetbrains.com/python/pycharm-community-#{version.after_comma}.dmg"
   name 'PyCharm Community Edition'
   homepage 'https://confluence.jetbrains.com/display/PYH/JetBrains+PyCharm+Preview+(EAP)'
   license :apache
 
   conflicts_with cask: 'pycharm-ce'
 
-  app 'PyCharm CE 5.1 EAP.app'
+  app "PyCharm CE #{version.major_minor_patch} EAP.app"
```
Comment (vitorgalvao) [inline Casks/pycharm-ce-eap.rb]: Do you want ` #{version.after_comma.major_minor_patch}`?

File: Casks/pycharm-eap.rb [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
-  url "https://download.jetbrains.com/python/pycharm-professional-#{version}.dmg"
+  url "https://download.jetbrains.com/python/pycharm-professional-#{version.after_comma}.dmg"
   name 'PyCharm'
   homepage 'https://confluence.jetbrains.com/display/PYH/JetBrains+PyCharm+Preview+(EAP)'
   license :commercial
 
   conflicts_with cask: 'pycharm'
 
-  app 'PyCharm 5.1 EAP.app'
+  app "PyCharm #{version.major_minor_patch} EAP.app"
```
Comment (vitorgalvao) [inline Casks/pycharm-eap.rb]: Do you want `#{version.after_comma.major_minor_patch}`?

Comment (vitorgalvao) [inline Casks/pycharm-eap.rb]: Should be `#{version.before_comma}`.

File: Casks/pycharm-ce-eap.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/pycharm-ce-eap.rb]: Should be `#{version.before_comma}`.

**Comment by tjanson** [comment]
Updated as per comments.

**Pull request merged**